.github/workflows/docker.yml

```diff
@@ -31,6 +31,15 @@ jobs:
       - name: Build ROS workspace and run tests
         run: |
           docker compose run build_and_test
+      - name: Convince tests
+        run: |
+          docker compose run build_and_test diff -r \
```

Comment:
I don't think this is going to run the command you want... it might just run what is in the docker compose service and ignore the remaining command?

Probably best to make a single bash script that runs all the tests we want, and then put that as the `build_and_test` service

Reply:
Wait, that did work... interesting! When I get home I will look in more detail.

Reply:
i just asumed it would work like docker run ..